engine/execution/rpc/engine.go

```diff
@@ -545,7 +545,7 @@ func (h *handler) GetAccountAtBlockID(
 
 	value, err := h.engine.GetAccount(ctx, flowAddress, blockFlowID)
 	if err != nil {
-		if errors.Is(err, scripts.ErrStateCommitmentPruned) {
+		if errors.Is(err, state.ErrStateCommitmentPruned) {
```

Comment:
I wonder if we should distinguish between block not executed error, so that we can remove L539, since GetAccount is already checking that.